src/modules/helpers/index.ts

```diff
@@ -445,6 +445,50 @@ export class HelpersModule {
     return array[index];
   }
 
+  /**
+   * Returns a weighted random element from the given array. Each element of the array should be an object with two keys `weight` and `value`.
+   * Each `weight` key should be a number representing the probability of selecting the value, relative to the sum of the weights. Weights can be any positive float or integer.
+   * Each `value` key should be the corresponding value.
+   * For example, if there are two values A and B, with weights 1 and 2 respectively, then the probability of picking A is 1/3 and the probability of picking B is 2/3.
```

Comment:
<img src="https://user-images.githubusercontent.com/7195563/209955726-878aac13-7378-4cc7-ba39-e25e849a785a.png" width="300px"/>


When hovering over these JSDoc it is very long due to it is rendered as markdown which would need e.g. double-whitespaces at the end to simulate a newline

But instead of doing that and it will get removed by prettier we should maybe use ul-li for this

```suggestion
   * Returns a weighted random element from the given array. Each element of the array should be an object with two keys `weight` and `value`.
   *
   * - Each `weight` key should be a number representing the probability of selecting the value, relative to the sum of the weights. Weights can be any positive float or integer.
   * - Each `value` key should be the corresponding value.
   * 
   * For example, if there are two values A and B, with weights 1 and 2 respectively, then the probability of picking A is 1/3 and the probability of picking B is 2/3.
```

<img src="https://user-images.githubusercontent.com/7195563/209956052-5c7793e9-a15b-41c9-85f0-99a097baa58a.png" width="300px"/>

This makes it much more readable